common/commands/curl_test.go

```diff
@@ -102,3 +102,106 @@ func TestBuildCommandUrl(t *testing.T) {
 		})
 	}
 }
+
+func TestFindUriWithStandaloneFlags(t *testing.T) {
+	tests := []struct {
+		name             string
+		arguments        []string
+		expectedUriIndex int
+		expectedUri      string
+	}{
+		{
+			name:             "test1",
```

Comment:
having a short description for each test will help to figure out scenario instead of naming it as test1,2 etc